addons/website_partner/views/website_partner_templates.xml

```diff
@@ -15,17 +17,25 @@
 </template>
 
 <template id="partner_detail" name="Partner Details">
-    <div class="row" itemscope="itemscope" itemtype="http://schema.org/Organization">
+    <div class="row mt-3" itemscope="itemscope" itemtype="http://schema.org/Organization">
```

Comment:
(seems it wasn't submitted somehow)

Just note this schema is not correct if "contact_details" is not enabled.
I don't think it's worth conditionally removing it but worth pointing out.